src/parser/mod.rs

```diff
@@ -15375,6 +15391,17 @@ impl<'a> Parser<'a> {
     }
 }
 
+fn prefixed_expr(expr: Expr, prefix: Option<Ident>) -> Expr {
```

Comment:
```suggestion
fn maybe_prefixed_expr(expr: Expr, prefix: Option<Ident>) -> Expr {
```
thinking in order to flag that the prefixing is optional